doT.js

```diff
@@ -24,7 +24,8 @@
 			doNotSkipEncoded: false
 		},
 		template: undefined, //fn, compile template
-		compile:  undefined  //fn, for express
+		compile:  undefined, //fn, for express
+		log: false
```

Comment:
docs are here at the moment: https://github.com/olado/olado.github.com/blob/master/doT/index.html